Source/automap.h

```diff
@@ -77,10 +88,15 @@ void AutomapZoomOut();
  */
 void DrawAutomap(const Surface &out);
 
+/**
+ * @brief Updates automap explorer at point if value is higher than existing.
+ */
+void UpdateAutomapExplorer(Point map, MapExplorationType explorer);

```

Comment:
A `map` is not a `Point`. I suggest renaming the argument:
Also suggest renaming the method to reflect the enum:
```suggestion
void UpdateAutomapExplorationState(Point mapPosition, AutomapTileExplorationState explorationState);
```

Reply:
Your point is invalid
![image](https://user-images.githubusercontent.com/14297035/135448010-f8e12a32-ab20-43db-992a-1c8f87cccef5.png)
xD